Gems/ScriptCanvas/Code/Editor/View/Windows/Tools/UpgradeTool/FileSaver.cpp

```diff
@@ -8,20 +8,24 @@
 
 #include <AzCore/Asset/AssetManagerBus.h>
 #include <AzCore/Component/TickBus.h>
-#include <AzCore/IO/SystemFile.h>
 #include <AzCore/IO/IStreamer.h>
 #include <AzCore/IO/Streamer/FileRequest.h>
+#include <AzCore/IO/SystemFile.h>
 #include <AzFramework/Asset/AssetSystemBus.h>
 #include <AzFramework/IO/FileOperations.h>
 #include <AzToolsFramework/API/EditorAssetSystemAPI.h>
 #include <AzToolsFramework/SourceControl/SourceControlAPI.h>
-#include <Editor/View/Windows/Tools/UpgradeTool/FileSaver.h>
 #include <ScriptCanvas/Assets/ScriptCanvasFileHandling.h>
+#include <ScriptCanvas/Core/GraphSerialization.h>
+
+#include <Editor/View/Windows/Tools/UpgradeTool/FileSaver.h>
 
 namespace ScriptCanvasEditor
 {
     namespace VersionExplorer
     {
+        using namespace ScriptCanvas;
+
```

Comment:
Repeat: the `using namespace ScriptCanvas;` call needs to be moved to function scope or removed altogether